marimbabot_description/urdf/marimbabot_ur5.urdf.xacro

```diff
@@ -128,13 +134,36 @@
   </joint>
 
   <xacro:if value="$(arg include_mallet)">
-    <xacro:include filename="$(find marimbabot_description)/urdf/static_mallet_holder.urdf" />
+    <xacro:if value="${mallet_holder_type_param == 'static'}">
```

Comment:
can't we use `$(arg mallet_holder_type) == 'static'` here directly instead of setting a param?